docs/source/quickstart.rst

```diff
@@ -29,8 +29,8 @@ Dask-Gateway can be installed with ``conda`` or ``pip``. It's composed of two pa
     $ pip install dask-gateway dask-gateway-server
 
 
-Start the gateway server
-------------------------
+1) Start the gateway server
+---------------------------
```

Comment:
Why the numbering?

Reply:
My thought was the numbering clearly indicated steps: 1, 2, etc.  I can remove them if don't like them.